deploy/examples/operator.yaml

```diff
@@ -668,10 +646,9 @@ spec:
           #  requests:
           #    cpu: 200m
           #    memory: 128Mi
-
-          #  Uncomment it to run lib bucket provisioner in multithreaded mode
-          #- name: LIB_BUCKET_PROVISIONER_THREADS
-          #  value: "5"
+      #  Uncomment it to run lib bucket provisioner in multithreaded mode
```

Comment:
The changes are too extensive in all the yamls, we would like to keep the indentation as it was for all these comments in the examples. Agreed with Blaine, if we can just find the minimal changes.